lib/hooks/index.js

```diff
@@ -114,6 +118,16 @@ function activate(agent) {
     moduleHook.patch(instrumentation.module, agent, instrumentation.version);
   }
 
+  for (var hookName in toInstrument) {
+    var regex = new RegExp(path.sep + hookName + path.sep);
+    for (var file in require.cache) {
+      if (file.match(regex)) {
+        logger.warn('Module ' + hookName + ' was loaded before trace agent.');
```

Comment:
It might be better to print file instead of regex. Given the way this code is written, you are going to have false positives.

Alternatively, you may want to grep for, e.g. `node_modules/redis/` to be a bit more precise.